pycds/__init__.py

```diff
@@ -158,8 +173,10 @@ class Obs(Base):
     history = relationship("History", backref=backref('obs_raw', order_by=id))
     variable = relationship(
         "Variable", backref=backref('obs_raw', order_by=id))
-    flags = relationship(
-        "NativeFlag", secondary=association_table, backref="flagged_obs")
+    flags = relationship("NativeFlag", secondary=obs_raw_native_flags_t, backref="flagged_obs")
+    # better named alias for 'flags'; don't repeat backref
+    native_flags = relationship("NativeFlag", secondary=obs_raw_native_flags_t)
```

Comment:
I left the original `flags` attribute because it seemed likely that other code depended on it and I didn't want to break it. I can search for usages and fix that, but at the time I wrote this I had enough to do without refactoring other code. Would you prefer that?